spec/status.yml

```diff
@@ -0,0 +1,44 @@
+components:
+  schemas:
+    Authenticators:
+      type: object
+      properties:
+        installed:
+          description: "The authenticators installed on the Conjur server"
+          type: array
+          items:
+            type: string
+          example: [ "authn" ]
+
+        configured:
+          description: "The authenticators configured on the Conjur server"
+          type: array
+          items:
+            type: string
+          example: [ "authn" ]
+
+        enabled:
+          description: "The authenticators enabled on the Conjur server"
+          type: array
+          items:
+            type: string
+          example: [ "authn" ]
+
+  paths: 
+    ListAuthenticators:
+      get:
+        tags:
+        - "status"
+        summary: "Details about which authenticators are on the Conjur Server"
+        description: |
+          "Response contains three members: installed, configured, and enabled. which tell
```

Comment:
i don't want to comment on the way we phrase this as you know better than me the language but just want to clarify:
- installed: The authenticator is implemented in Conjur and is available for configuration
- configured: The authenticator has a webservice in the DB that was loaded by policy
- enabled: The authenticator is enabled (in the DB or in the ENV) and is ready for authentication

Reply:
@izgeri ^

btw what i refer to as the authenticator status api is this: https://docs.conjur.org/Latest/en/Content/Integrations/Authn-status.htm#Enable

this endpoint returns the status of a specific authenticator.

Reply:
i would refer to the api described in this page as "authenticators index" to reduce confusion